src/main/java/com/redhat/devtools/intellij/lsp4ij/LanguageServerWrapper.java

```diff
@@ -793,7 +793,7 @@ private CompletableFuture<LanguageServer> connect(@Nonnull URI absolutePath, Doc
 
         VirtualFile file = FileDocumentManager.getInstance().getFile(document);
         if (file != null && file.exists()) {
-            watchProject(LSPIJUtils.getProject(file), false);
+            watchProject(LSPIJUtils.getModule(file), false);
```

Comment:
maybe rename to watchModule

Reply:
I remove this code since we don't support workspace folder.